.nvmrc

```diff
@@ -0,0 +1 @@
+v16
```

Comment:
you may as well also add this to the `package.json` as the engine. on my phone rn so i can't really do anything other than suggest